demo/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: demo
+---
+
+<div style="margin: 0px 1em 2em 1em;">
+On this page you can play with a live demo of the Cryostat Web UI. This runs entirely in your browser and does not access any real Cryostat backend server instances.
+Core JFR functionality is not available in this stubbed out demo, but you can use this to get a sense of what features Cryostat offers and what it looks like.
+</div>
+
+<div style="position: relative;">
+  <div
+    style="position: absolute; left: 0; top: 0; width: 100%; height: 100%; align-items: center; display: flex; justify-content: center">
+      <div id="loading" aria-label="Loading demo" class="linear-dot-spinner"></div>
```

Comment:
Sorry, the id should on the parent or else the view won't be interact-able. I thought i updated the diff but didnt.

```diff
 <div style="position: relative;">
   <div
+    id="loading"
     style="position: absolute; left: 0; top: 0; width: 100%; height: 100%; align-items: center; display: flex; justify-content: center">
-      <div id="loading" aria-label="Loading demo" class="linear-dot-spinner"></div>
+      <div aria-label="Loading demo" class="linear-dot-spinner"></div>
   </div>
```